web/src/components/merchant/sign-up/SignUpCard.tsx

```diff
@@ -118,24 +118,50 @@ const SignUpCard: React.FC = () => {
     // If both passwords are empty we show PASSWORD_EMPTY.
     event.target.value === password
       ? setConfirmPasswordError('')
-      : setConfirmPasswordError(PASSWORDS_DO_NOT_MATCH);
+      : setConfirmPasswordError(Errors.PASSWORDS_DO_NOT_MATCH);
     setConfirmPassword(event.target.value);
   };
 
   const handleVpaChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    event.target.value ? setVpaError('') : setVpaError(VPA_EMPTY);
+    event.target.value ? setVpaError('') : setVpaError(Errors.VPA_EMPTY);
     setVpa(event.target.value);
   };
 
   const handleSubmit = () => {
```

Comment:
can you disable the submit button if there is some error

Reply:
Okay!